using_images/s2i_images/ruby.adoc

```diff
@@ -117,6 +124,40 @@ link:../../dev_guide/environment_variables.html#set-environment-variables[the
 $ oc env dc/rails-app RAILS_ENV=development
 ----
 
+== Performance tuning
+You can tune the number of threads per worker using the
+`*PUMA_MIN_THREADS*` and `*PUMA_MAX_THREADS*` environment variables.
+Additionally, the number of worker processes is determined by the number of CPU
+cores that the container has available, as recommended by
+link:https://github.com/puma/puma[Puma]'s documentation. This is determined using
+the cgroup link:https://www.kernel.org/doc/Documentation/cgroup-v1/cpusets.txt[cpusets]
+subsystem. You can specify the cores that the container is allowed to use by passing
+the `--cpuset-cpus` parameter to the link:http://docker.io[Docker] run command:
```

Comment:
we shouldn't be talking about docker run'ing images in the openshift-docs.  we should only doc things that are relevant to using the image in openshift.